### PR TITLE
compsupp-6747

### PR DESCRIPTION
**Pull request by diiegopereira**
compsupp-6747 - Updated XML config for Beaver Builder

### CODE REVIEW

**Comment by diiegopereira** [comment]
Sorry, I created a new folder because I thought it used the folder slug to do the sync.
I think I found the problem now, the plugin name that we used to test was "Beaver Builder Plugin (Agency Version)".
I deleted the "bb-folder" and update our index to also include this version.

File: beaver-builder/wpml-config.xml [hunk 1]
```diff
@@ -1,32 +1,28 @@
...
-	</custom-fields>
-	<beaver-builder-widgets>
-		<widget name="button-group">
-			<fields-in-item items_of="items">
-				<field type="Button label" editor_type="LINE">text</field>
-				<field type="Button link" editor_type="LINK">link</field>
-				<field type="Lightbox HTML" editor_type="AREA">lightbox_content_html</field>
-			</fields-in-item>
-		</widget>
-		<widget name="icon-group">
```
Comment (strategio) [inline beaver-builder/wpml-config.xml:16]: @diiegopereira, why did you remove the config for `icon-group`?

Even if this widget does not exist in the latest version, we should keep it for backward compatibility.

Comment (diiegopereira) [reply beaver-builder/wpml-config.xml:16]: Sorry, it was removed by mistake. I added it again now.

**Comment by strategio** [comment]
Merged and deployed @diiegopereira.
